pkgs/os-specific/linux/tomb/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, zsh, pinentry, cryptsetup, gnupg1orig, makeWrapper }:
+
+let
+    version = "2.2";
+in
+
+stdenv.mkDerivation rec {
```

Comment:
It makes no real difference, but the `rec` here is strictly unnecessary, as you're not making use of the recursion. Usually, `version` is defined within the attrset being passed to `mkDerivation`. `rec` is then used to allow referring to `version` within `name`.